### PR TITLE
49de51 from proper branch

### PR DESCRIPTION
**Pull request by Koeng101**
Pull request for 49de51 from proper branch

For meet at 11:30. Need to fix:

Heights with magdeck
Anything other small modifications I should know about

### CODE REVIEW

File: protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py [hunk 1]
```diff
@@ -1,21 +1,18 @@
-from time import sleep
+import math
+from opentrons.types import Point
 
 metadata={"apiLevel": "2.3"}
```
Comment (ncdiehl11) [inline protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py:4]: please add `protocolName`,  `author`, and `source` fields. example:
```
metadata = {
    'protocolName': 'Fix & DAPI Staining',
    'author': 'Chaz <chaz@opentrons.com>',
    'source': 'Custom Protocol Request',
    'apiLevel': '2.5'
}
```

File: protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py [hunk 2]
```diff
@@ -70,126 +72,94 @@ def pick_up_plate():
     
     def initial_lysis(plate):
         pick_up_plate()
-        [p300m.transfer(500, lysis_buffer_a_wash_1, col) for col in plate.columns()]
+        [p300m.transfer(500, lysis_buffer_a_wash_1, col) for col in plate_cols]
```
Comment (ncdiehl11) [inline protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py:75]: A general stylistic note for protocols: list comprehensions are great for variable assignments (i.e. accessing wells of a plate in some order), but they can be problematic for API commands like `transfer`. If we need to troubleshoot later and break the `transfer` into multiple atomic commands, we'll need to change the list comp to a loop

File: protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py [hunk 3]
```diff
@@ -1,21 +1,18 @@
-from time import sleep
+import math
```
Comment (ncdiehl11) [inline protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py:1]: Please make sure you run `make all` to create the `protoBuilds` files that will build the custom protocol site

**Review by ncdiehl11 — DISMISSED** [review]

File: protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py (hunk 3 above)
Comment (Koeng101) [reply protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py:1]:
```
[koeng@kgnode2 Protocols]$ make all
mkdir -p protoBuilds/6f4e2c/
source venvs/ot2/bin/activate && \
export OVERRIDE_SETTINGS_DIR=ot2monorepoClone/api/tests/opentrons/data && \
python protolib/parse/parseOT2v2.py protocols/6f4e2c/6f4e2c.ot2.apiv2.py protoBuilds/6f4e2c/6f4e2c.ot2.apiv2.py.json && \
deactivate
/bin/bash: venvs/ot2/bin/activate: No such file or directory
make: *** [Makefile:55: protoBuilds/6f4e2c/6f4e2c.ot2.apiv2.py.json] Error 1
```

I got the following error when running `make all` from the Protocols root directory

File: protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+metadata = {
+        'apiLevel': '2.5',
+        'protocolName': 'ThermoFisher MagMAX Plant DNA Isolation',
+        'author': 'Chaz <chaz@opentrons.com>',
+        'source': 'Custom Protocol Request'
+        }
+
+
+def run(ctx):
+
```
Comment (chazmanc) [inline protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py:13]: @Koeng101 there should be a line here that looks like this:
```
[num_samples] = get_values(  # noqa: F821
'num_samples')
```
the variable `num_samples` can then be accessed and will be an integer based on your fields.json

Comment (Koeng101) [reply protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py:13]: Gotcha, another push going with that

**Comment by chazmanc** [comment]
@Koeng101 I think the most recent issue (`make build` failing) is coming up because you didn't build the files for the website. By successfully running `make all`, you should build a protocol file, metadata file, and a readme file (all .json files) that will be housed in an identical folder in **protoBuilds**

**Comment by Koeng101** [comment]
At the end of the `make all` command, I get the following error:
```
Traceback (most recent call last):
  File "protolib/traverse_errors.py", line 120, in <module>
    write_metadata_to_file(PROTOCOL_DIR)
  File "protolib/traverse_errors.py", line 115, in write_metadata_to_file
    'status': get_status(metadata)},
  File "protolib/traverse_errors.py", line 86, in get_status
    errors = get_errors(file_data['files'])
  File "protolib/traverse_errors.py", line 67, in get_errors
    raise ValueError('Found 0 protocol files required at least 1')
ValueError: Found 0 protocol files required at least 1
make: *** [Makefile:46: parse-errors] Error 1
```

I decided to check protoBuilds/49de51 directory, and found only the protocol json. Seems like something is going wrong, but I can't tell what....

**Review by chazmanc — APPROVED** [review]

**Pull request merged**